src/dotnet/commands/dotnet-sln/LocalizableStrings.cs

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DotNet.Tools.Sln
+{
+    internal class LocalizableStrings
+    {
+        public const string AppFullName = ".NET modify solution file command";
+
+        public const string AppDescription = "Command to add, remove, and list projects from the solution (SLN) file.";
+
+        public const string AppHelpText = "Projects to add to the solution.";
+
+        public const string CmdSlnFile = "<SLN_FILE>";
+        
+        public const string AddAppFullName = ".NET Add project to a solution file Command";
```

Comment:
For the add and remove subcommands, if you pass multiple projects we will try to add|remove all that you specify. And if any fails the whole operation fails. Do we want to re-word? Or is it ok to make it seem like you can only add|remove one project at a time?